scmtfvc.properties

```diff
@@ -2,12 +2,19 @@ category=Integration
 description=Provides SCM TFVC integration
 devVersion=
 privateVersions=
-publicVersions=2.4.0
-archivedVersions=
+publicVersions=2.5.0
+archivedVersions=2.4.0
 
 defaults.mavenGroupId=org.sonarsource.sonar-plugins.tfvc
 defaults.mavenArtifactId=sonar-scm-tfvc-plugin
 
+2.5.0.description=update DevOps api
+2.5.0.sqs=[10.8,LATEST]
+2.5.0.sqcb=[24.12,LATEST]
+2.5.0.downloadUrl=https://github.com/lg2de/sonar-scm-tfvc/releases/download/2.5.0/sonar-scm-tfvc-plugin-2.5.0.318.jar
+2.5.0.date=2025-05-05
+2.5.0.changelogUrl=https://github.com/lg2de/sonar-scm-tfvc/releases/tag/2.5.0
+
 2.4.0.description=update DevOps api
 2.4.0.sqVersions=[9.9,10.7]
 2.4.0.sqs=[10.8,LATEST]
```

Comment:
This and the next line need to be changed. Archived versions can't be compatible with `LATEST`, and only one version can be compatible with `LATEST` at a time. You should set these end ranges to the latest versions, currently 2025.2 for `sqs` and 25.5 for `sqcb`

Reply:
Even I do not fully understand, why older versions cannot be compatible with LATEST (because they are!), I've changed this.